zerver/lib/message.py

```diff
@@ -1197,16 +1197,16 @@ def aggregate_streams(*, input_dict: Dict[int, RawUnreadStreamDict]) -> List[Unr
 
 
 def aggregate_pms(
```

Comment:
Presumably `aggregate_pms` is a private-message reference, too?

Reply:
Yeah, there will need to be a pass for lowercase `pm` in the codebase. I tackled the uppercase instance ("PM") here because those are almost all code comments or doc-strings.